jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcMetadataLoader.java

```diff
@@ -92,20 +91,27 @@ public void loadTables(JdbcSchema schema, Connection connection) {
         }
     }
 
+    private String getSchemaName(Schema schema) {
```

Comment:
This method name here can be a bit confusing because there is both a concept of "schema" in JDBC and in MetaModel. And the unfortunate thing is that they aren't 100% aligned, which is basically what the _usesCatalogAsSchemas flag is used to determine.

I would rename it to "getJdbcSchemaName" to avoid that confusion.